### DISCUSSION TITLE
how to generate a exe stager?

### DISCUSSION THREAD

**Discussion by YeBottle** [BishopFox/sliver#1012] [General]
i have generate shellcode ,and put it into c , then build it to an exe file, run it successful in windows,
but our leader ask me to generate an exe file directly, for he knows that profile support -f exe ,
i tried many times but got no ideas; 
can anyone give some suggestion?

**Comment by moloch--** [comment]
We have some examples in the wiki:
https://github.com/BishopFox/sliver/wiki/Stagers
